src/victory-theme/material.js

```diff
@@ -168,27 +182,37 @@ export default {
           stroke: "transparent",
           strokeWidth: 0,
           textAnchor: "middle"
-        }),
-      parent: {}
-    }
+        })
+    },
+    props: baseProps
```

Comment:
Do the props need to be namespaced under `props`? it would be nice if it could just match the whole props object.